src/test/java/spoon/test/query_function/VariableReferencesTest.java

```diff
@@ -196,8 +196,6 @@ public void testVariableScopeFunction() {
 				CtElement[] real = var.map(new VariableScopeFunction()).list().toArray(new CtElement[0]);
 				if(var instanceof CtLocalVariable) {
 					assertArrayEquals(var.map(new LocalVariableScopeFunction()).list().toArray(new CtElement[0]), real);
-				} else if(var instanceof CtField) {
-					assertArrayEquals(var.map(new FieldScopeFunction()).list().toArray(new CtElement[0]), real);

```

Comment:
@martinezmatias a rotten assertion

Reply:
haha great!! Nice finding. I will run the rotten detection to know whether it is detected :)